Sources/_StringProcessing/Engine/Registers.swift

```diff
@@ -120,11 +130,15 @@ extension Processor.Registers {
 
     self.values = Array(
       repeating: SentinelValue(), count: info.values)
+    self.positions = Array(
+      repeating: Processor.Registers.sentinelIndex,
+      count: info.positions)
```

Comment:
Future: Why the start index instead of the provided sentinel? Should or could we construct an invalid index and at least assert that we're never loading that?

Reply:
`SentinelValue` only works on the value registers since they are `[Any]`. For now we only emit position registers in this one case, if/when we have more complicated cases in the future we'll want some kinda validation like that